eslzArm/managementGroupTemplates/policyAssignments/DINE-ResourceDiagnosticsPolicyAssignment.json

```
@@ -29,15 +29,15 @@
     },
     "variables": {
         "policyDefinitions": {
-            "deployResourceDiagnostics": "[concat('/providers/Microsoft.Management/managementGroups/', parameters('topLevelManagementGroupPrefix'), '/providers/Microsoft.Authorization/policySetDefinitions/Deploy-Diagnostics-LogAnalytics')]"
+            "deployResourceDiagnostics": "/providers/Microsoft.Authorization/policySetDefinitions/0884adba-2312-4468-abeb-5422caed1038"
         },
         "policyAssignmentNames": {
             "resourceDiagnostics": "Deploy-Resource-Diag",
```

```suggestion
            "resourceDiagnostics": "Deploy-Diag-Logs",
```

can we change to this, so it'll trigger a TF change as this one already exists in TF?